packages/arch/packages/tooltip/src/index.js

```diff
@@ -40,7 +40,7 @@ let TooltipPositioner = (props: PositionerProps) => {
     <Popper
       referenceElement={props.targetNode}
       placement={props.placement}
-      modifiers={{ preventOverflow: { enabled: false } }}
+      modifiers={{ hide: { enabled: false }, preventOverflow: { enabled: false } }}
```

Comment:
Without this, I get lots of `'preventOverflow' modifier is required by 'hide' modifier in order to work, be sure to include it before 'hide'!` warnings from `popper.js`.